data/mse-v2.json

```diff
@@ -0,0 +1,19 @@
+{
+  "url": "https://wicg.github.io/media-source/",
+  "title": "Media Source Extensions™",
+  "wgs": [
+    {
+      "url": "https://www.w3.org/community/wicg/",
+      "label": "Web Platform Incubator Community Group"
+    }
+  ],
+  "features": {
+    "codec-switching": {
+      "title": "Codec switching",
+      "impl": {
+        "chromestatus": 5719220952236032
```

Comment:
Here's the Firefox bug for this feature: https://bugzilla.mozilla.org/show_bug.cgi?id=1481166